todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/activities/detail/DetailActivity.java

```diff
@@ -150,12 +198,12 @@ private void displayDetail() {
         dateSwitch.setChecked(todoItem.hasDate());
     }
 
-    private ToDoContent getTodo() throws AssertionError {
+    private ToDoContent getTodo() throws Exception {
```

Comment:
we use to use AssertionError here to catch when the intent had no extras, this case is now managed through creating a new ToDo